iModelCore/ECPresentation/Source/Content/ContentSpecificationsHandler.h

```diff
@@ -255,8 +256,9 @@ struct ContentSpecificationsHandler
     private:
         ExpressionContextPtr m_expressionContext = nullptr;
     protected:
-        virtual bool _Supports(ECPropertyCR, PropertySpecificationsList const&) = 0;
-        virtual PropertyAppendResult _Append(ECPropertyCR, Utf8CP, PropertySpecificationsList const&) = 0;
+        virtual bool _Supports(ECPropertyCR, ECClassCR, PropertySpecificationsList const&) = 0;
+        virtual PropertyAppendResult _Append(ECPropertyCR, ECClassCR, Utf8CP, PropertySpecificationsList const&) = 0;
+        virtual PropertyAppendResult _AppendCalculatedProperty(ECClassCP ecClass, CalculatedPropertiesSpecificationCP spec, Utf8StringCR label, Utf8StringCR name) { return PropertyAppendResult(false); };

```

Comment:
- can't we get label and name from `spec`?
- shouldn't spec be a `CR`?

Reply:
fixed